src/components/Content.tsx

```diff
@@ -167,48 +167,69 @@ const Content = ({...props}: ContentProps) => {
-            if (currentForm.data && currentForm.data.formByCreatedAt.items[0]) {
-                if (typeof formDefPaginated === 'undefined') {
-                    formDefPaginated = currentForm.data.formByCreatedAt.items[0];
-                    questions = currentForm.data.formByCreatedAt.items[0].questions.items;
-                } else {
-                    questions = questions.concat(currentForm.data.formByCreatedAt.items[0].questions.items);
+        try {
+            do {
+                let currentForm: any = await helper.callGraphQL<FormDefinitionByCreatedAtPaginated>(customQueries.formByCreatedAtPaginated, {...customQueries.formByCreatedAtInputConsts, nextToken: nextToken});
+                if (currentForm.data && currentForm.data.formByCreatedAt.items[0]) {
+                    if (typeof formDefPaginated === 'undefined') {
+                        formDefPaginated = currentForm.data.formByCreatedAt.items[0];
+                        questions = currentForm.data.formByCreatedAt.items[0].questions.items;
+                    } else {
+                        questions = questions.concat(currentForm.data.formByCreatedAt.items[0].questions.items);
+                    }
+                    nextToken = currentForm.data.formByCreatedAt.items[0].questions.nextToken;
                 }
-                nextToken = currentForm.data.formByCreatedAt.items[0].questions.nextToken;
+            } while (nextToken);
+            
+            if (formDefPaginated) {
+                let formDef: FormDefinition = {
+                        id: formDefPaginated.id,
+                        createdAt: formDefPaginated.createdAt,
+                        questions: {
+                            items: questions
+                        }
+                    };
+                    console.log("FormDef:", formDef);
+                    setFormDefinition(formDef);
+                    let quAns = createQuestionAnswers(formDef);
+                    let userAnswers = await getUserAnswers(formDef);
+                    setFirstAnswers(quAns, userAnswers);
+            } else {
+                console.log("Error loading form definition!");
             }
-        } while (nextToken);
-        if (formDefPaginated) {
-            let formDef: FormDefinition = {
-                    id: formDefPaginated.id,
-                    createdAt: formDefPaginated.createdAt,
-                    questions: {
-                        items: questions
-                    }
-                };
-                console.log("FormDef:", formDef);
-                setFormDefinition(formDef);
-                let quAns = createQuestionAnswers(formDef);
-                let userAnswers = await getUserAnswers();
-                setFirstAnswers(quAns, userAnswers);
-        } else {
-            console.log("Error loading form definition!");
+        } catch(e) {
+            console.error("GraphQL error while fetching form definition and user answers!");
```

Comment:
Kanskje legge til e i console.log?
```suggestion
            console.error("GraphQL error while fetching form definition and user answers! ", e);
```